plugin-api/src/main/java/org/hyperledger/besu/plugin/services/storage/PrivacyKeyValueStorageFactory.java

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+package org.hyperledger.besu.plugin.services.storage;
+
+public interface PrivacyKeyValueStorageFactory extends KeyValueStorageFactory {
+  /**
+   * Retrieves the schema version of the key-value storage factory.
+   *
+   * @return the schema version of the key-value storage factory.
+   */
+  int getSchemaVersion();
```

Comment:
Could that be more than the "schema" version? I think database version would be a better name ...

Reply:
What about just a generic `getVersion()` and a better comment?